### PR TITLE
Small map fixes

### PR DESCRIPTION
**Pull request by PrimeGenerations**
# What this PR does

Fixes being able to use dropship from outstide.

Makes Charlie Airlock immortal. Bye bye bubblegum corpse destroyer of airlocks.

# Why it should be added to the game

Fixes

### CODE REVIEW

**Comment by skigear** [comment]
being able to get on the shuttle's back before it takes off isn't really an issue, other than that it can make getting on it easier

adding this would also get rid of jumping into bluespace, which is a mechanic that also doesn't cause much issue

File: maps/core/burgerstation.dmm [hunk 1]
```diff
@@ -2878,8 +2891,11 @@
 /turf/simulated/floor/tile/grey,
 /area/burgerstation/hall/north)
 "ku" = (
-/turf/simulated/floor/plating,
-/area/transit/dropship/bravo/station)
+/obj/structure/interactive/restocker/ammo{
+	anchored = 1
```
Comment (BurgerLUA) [inline maps/core/burgerstation.dmm:2895]: Wait, why is this anchored?

File: maps/core/burgerstation.dmm [hunk 2]
```diff
@@ -205,16 +205,19 @@
 /turf/simulated/floor/tile/dark,
 /area/burgerstation/armory)
 "aO" = (
-/turf/simulated/floor/plating/shuttle,
-/area/transit/dropship/delta/station)
+/obj/structure/interactive/restocker/ammo{
+	anchored = 1
```
Comment (BurgerLUA) [inline maps/core/burgerstation.dmm:209]: this too

**Review by BurgerLUA — CHANGES_REQUESTED** [review]
What specifically did you do to make the airlock immortal? Also why are the ammo restockers anchored now?

**Comment by BurgerLUA** [comment]
PR is 22 days stale with conflicting files, so I'm closing it.

**Pull request closed without merging**